package.json

```diff
@@ -29,9 +29,13 @@
   },
   "devDependencies": {
     "broccoli": "^0.16.9",
+    "broccoli-fixture": "^1.0.0",
     "codecov": "^1.0.1",
     "istanbul": "^0.4.1",
-    "mocha": "~3.0.2",
+    "mocha": "~8.2.0",
     "walk-sync": "~0.1.2"
+  },
+  "engines": {
+    "node": "10.* || >= 12"
```

Comment:
Can you split the "drop old node support" stuff into a separate PR? It will make changelog/reviewing much easier to have separate conceptually atomic commits.

Reply:
Yep. The problem is that the test PR won't pass CI without newer node. But I'll make a separate PR and make the test changes dependent on the node one.

Reply:
https://github.com/rickharrison/broccoli-asset-rewrite/pull/77